src/software/ai/hl/stp/play/timeout_play/timeout_play_fsm.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include "proto/parameters.pb.h"
+#include "shared/constants.h"
+#include "software/ai/hl/stp/play/play_fsm.h"
+#include "software/ai/hl/stp/tactic/halt/halt_tactic.h"
+#include "software/logger/logger.h"
+
+struct TimeoutPlayFSM
+{
+    struct ControlParams
+    {
+    };
+
+    class TimeoutState;
+
+    DEFINE_PLAY_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
+
+    /**
+     * Creates a TimeoutPlay FSM
+     *
+     * @param ai_config the play config for this FSM
+     */
+    explicit TimeoutPlayFSM();
```

Comment:
docs are wrong. also you can actually delete this code and let C++ generate the default constructor